app/forms/new-cluster.html

```diff
@@ -42,7 +42,7 @@ <h4 class="modal-title" translatable="yes">Deploy cluster</h4>
           <input id="cluster-workerconfig-name" class="form-control input-lg" type="text" ng-model="fields.workerconfigname"/>
         </div>
         <div class="form-group">
-          <label for="cluster-spark-image">Apache Spark image (only radanalyticsio/openshift-spark is supported)</label>
+          <label for="cluster-spark-image">Apache Spark image (default is SPARK_DEFAULT)</label>
```

Comment:
Will an end user know what "SPARK_DEFAULT" is?

Reply:
It's replaced with "sed" in launch.sh before the app is started, currently based on the value reported by the embedded CLI

Reply:
I think it would be nicer to make it an env var that's passed to server.js and fetched by angular at runtime for use.  IMO, it's slightly more in-line with the rest of the system

Reply:
On second thought, I'm ok with this.  Should be a short-lived compromise...and it works.